packages/devextreme/testing/tests/DevExpress.ui.widgets/gallery.tests.js

```diff
@@ -385,15 +385,15 @@ QUnit.module('behavior', {
         assert.equal($loopItems.length, 2);
     });
 
-    QUnit.test('duplicate items should have role="presentation"', function(assert) {
+    QUnit.test('duplicate items should be hidden for readres', function(assert) {
```

Comment:
[minor, test naming]
```suggestion
    QUnit.test('duplicate items should have aria-hidden=true', function(assert) {
```

Reply:
fixed